developer/handbook.md

```diff
@@ -519,14 +527,58 @@ Management APIs return the compacted form.
 }
 ```
 
-Similarly, the `odrl:or` could have been replaced with an `odrl:and` or `odrl:xone` (exclusive-or). 
-
+Similarly, the `odrl:or` could have been replaced with an `odrl:and` or `odrl:xone` (exclusive-or).
 
 #### Contract definitions
 
-##### Contract policy
+Contract definitions are how [assets](#assets) and [policies](#policies) are linked together. It is our way of
+expressing which policies are in effect for an asset. So when we want to offer an asset (or several assets) in the
+dataspace, we use a contract definition to express under what conditions we do that. Those conditions are comprised of a
+_contract policy_ and an _access policy_, see below for details. Those policies are referenced by ID, that means they
+must already exist in the policy store before creating the contract definition.
+
+It is important to note that contract definitions are a _internal objects_, i.e. they **never** leave the realm of the
+provider, and they are **never** sent to the consumer.
+
+- **access policy**: determines whether a particular consumer is offered an asset or not. For example, we may want to
+  restrict certain assets such that only consumers within a particular geography can see them. Consumers outside that
+  geography wouldn't even have them in their [catalog](#catalog).
+- **contract policy**: determines the conditions for initiating a contract negotiation for a particular asset. Note that
+  does not automatically guarantee the successful _creation_ of a contract, it merely expresses the _eligibility_ to
+  start the negotiation.
```

Comment:
Makes sense. I think I was a step too far, as there is not yet a mechanism for counter-offers, because then the policy may be modified during the negotiation.